src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAOAEPKeyExchangeFormatter.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    public class RSAOAEPKeyExchangeFormatter : AsymmetricKeyExchangeFormatter
+    {
+        private byte[] ParameterValue;
+        private RSA _rsaKey;
+
+        public RSAOAEPKeyExchangeFormatter() { }
+        public RSAOAEPKeyExchangeFormatter(AsymmetricAlgorithm key)
+        {
+            if (key == null)
+                throw new ArgumentNullException(nameof(key));
+
+            _rsaKey = (RSA)key;
+        }
+
+        /// <internalonly/>
```

Comment:
Sure will remove. I noticed that there's a bunch of others in our source so I left them originally.